less/navigation.less

```diff
@@ -1 +1,33 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.navbar {
+    position: fixed;
+    opacity: .7;
+    @media @phone {
+        width: 100%;
+    }
+    .flex-align(column, center);
+    width: 100%;
+    align-self: center;
+    background: @footer-background;
+    
+
+    h1 {
+        font-size: 3.5rem;
+
+    }
+
+    nav {
+        @media @phone {
+            width: 100%;
+        }
+        .flex-just(row, space-evenly);
+        width: 20%;
```

Comment:
I think you might want to change this width for the nav to 100%, it looks a bit squished.